src/components/Currencies.js

```diff
@@ -1,34 +1,94 @@
+/*
+		I added Radium to this component as well.
+		I decided to add CSS grid here in place of the custom grid I was using before.
+ */
+
 import React from 'react';
+import Radium from 'radium';
+import RadiumVars from '../RadiumVariables';
 
 const Currencies = (props) => {
 	const cryptoList = props.cryptoList;
 	const intlList = props.intlList;
-	return <div className="currencies">
-      <div className="row">
-        <div className="col-1-of-2">
-          <div className="currencies--center">
-            <p className="currencies__label">Cryptocurrencies</p>
-            <ul className="currencies__list">
+	return (
+		<div style={styles.currencies}>
+      <div style={styles.gridContainer}>
+        <div style={styles.gridItem}>
+          <div style={styles.currencies__center}>
+            <p style={styles.currencies__label}>Cryptocurrencies</p>
+            <ul style={styles.currencies__list}>
               {cryptoList.map(c => (
-                <li className="currencies__item" key={c.code}>
+                <li style={styles.currencies__item} key={c.code}>
                   {c.name} - {c.code}
                 </li>
               ))}
             </ul>
           </div>
         </div>
-        <div className="col-1-of-2">
```

Comment:
Main update here is that i added the use of CSS grid in place of the old hand written column setup.